### ISSUE TITLE
Custom Content fields and custom Date formats are not exported in the /csv/

### ISSUE THREAD

**Issue by rafaehlers** [gravityview/GravityView#1231]
A user reported and Toyin was able to confirm (https://debug.gravityview.co/testing/view/products/csv/):

```
I made a relatively basic table type view with personnalized content fields in it.
Some of these fields have static content and in some I used the gvlogic shortcode to add some logic on certain entrie values.
Here is an example of the logic applied:
[gvlogic if="{Type d'inscription:27}" is="Nouvel adhérent"]
Oui
[else]
Non
[/gvlogic]

I then checked the "Allow CSV acces" in the view options.

The view is working well, all the personnalized content fields are correctly displayed. But when I download the view in CSV format, then the personnalized content disappears. I simply get a blank field instead of the value.
This also happens with the date fields, if I set a specific format, it is respected in the view, but I get the basic database format once in the CSV format.
```

Tested with GV 2.2.3

**Comment by rafaehlers** [comment]
https://secure.helpscout.net/conversation/740212957/18345/

**Comment by soulseekah** [comment]
Fixed this particular issue.

This might pop up in later issues as well as CSV hasn't been tested too well. Problem is that all our current fields are HTML-specific. For everything else there's a fallback (field.php or field-csv.php for example). This fallback just outputs the stored value.

I just demoted content and date fields to generic fields (non HTML specific). They contain no HTML. We cannot do this with fields like ratings, since we don't want HTML markup to be exported in CSV. So right now those will be sent as they are stored.
